content/02.body.md

```diff
@@ -76,6 +76,14 @@ We use network permutation to isolate degree and determine its effects in differ
 Degree-preserving network permutation obscures true connections and higher-order connectivity information (e.g., community structure), while retaining node degree, and, thereby, the network's degree sequence.
 Thanks to the flexibility of permutation, our framework can quantify the effect of degree on any network edge prediction method.
 
+Several degree-preserving network permutation strategies have been developed including
+XSwap [@doi:10.1137/1.9781611972795.67],
+FANMOD (Fast Network Motif Detection) [@doi:10.1093/bioinformatics/btl038],
+CoMoFinder (Co-regulatory Motif Finder) [@doi:10.1093/bioinformatics/btv159],
+DIA-MCIS (Diaconis Monte Carlo Importance Sampling) [@doi:10.1093/bioinformatics/btm454],
+and WaRSwap (Weighted and Reverse Swap Sampling) [@doi:10.1186/gb-2013-14-8-r85].
+IndeCut proposed a method to characterize these strategies by their ability to uniformly sample from the solution space of all possible degree-preserving permutations [@doi:10.1093/bioinformatics/btx798].
```

Comment:
Following up on our conversation at https://github.com/greenelab/xswap-manuscript/issues/1#issuecomment-1753441938, @dkoslicki is this a faithful description of IndeCut and its relevance to degree-preserving permutation approaches?

Reply:
Noting that the "Method S2: Description of examined network motif discovery algorithms" from the [IndeCut supplement](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5925789/bin/btx798_supplementarydata.pdf) was super helpful in summarizing these methods.

Reply:
Yes, that does capture things nicely!